src/applyLookupTable.cc

```diff
@@ -38,10 +38,10 @@ int applyLookupTable(
     PixelT indOffset
 ) {
     int numOutOfRange = 0;
-    auto const maxLookupCol = table.size() - 1;
+    int const maxLookupCol = table.size() - 1;
```

Comment:
Easier to fix than to open a new ticket, so I'll just fix it here (by raising an exception if `table` is empty).